opentelemetry-api/src/opentelemetry/configuration/__init__.py

```diff
@@ -167,3 +179,20 @@ def _reset(cls) -> None:
         if cls._instance:
             cls._instance._config_map.clear()  # pylint: disable=protected-access
             cls._instance = None
+
+    def traced_request_attrs(self, instrumentation: str) -> List[str]:
```

Comment:
Nit: Can you make these protected? I believe we are only using these internally.

Reply:
Yeah sure. I was thinking we would let users use this if they need.